Solutions/RubrikSecurityCloud/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "publisherId": "rubrik_inc",
+    "offerId": "rubrik_sentinel",
+    "firstPublishDate": "2022-07-19",
+    "lastPublishDate": "2023-07-19",
+    "providers": "Rubrik",
+    "categories": {
+	"domains" : ["Security - Others"],
+	"verticals": []
+     },
+     "support": {
+         "name": "Rubrik",
+         "email": "support@rubrik.com",
+         "tier": "Partner-supported (Rubrik)",
```

Comment:
@benmeadowcroft, tier should be just "Partner". Please check and update, thanks.

Reply:
Thank you, have addressed in Thank you, have resolved in https://github.com/Azure/Azure-Sentinel/pull/6150/commits/f01f79705e6676807acffc5aaecd469db6f54990